routingv8/routes.go

```diff
@@ -0,0 +1,39 @@
+package routingv8
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Routes returns routes.
+// The required parameters for this resource is a origin location and a destination location.
+// See https://developer.here.com/documentation/routing-api/dev_guide/topics/send-request.html
+// for details about other parameters.
+func (s *RoutingService) Routes(
+	ctx context.Context,
+	req *RoutesRequest,
+) (_ *RoutesResponse, err error) {
+	u, err := s.URL.Parse("routes")
+	if err != nil {
+		return nil, err
+	}
+
+	values := make(url.Values)
+	values.Add("transportMode", "truck")
```

Comment:
I would suggest that we allow `transportMode` to be set in the request, similar to how it can be done for the matrix calculation. What do you think about that?

We will probably want to use `"car"` for this value, because we've noticed that HERE sometimes thinks `"truck"` cannot drive on routes we know our trucks can do.

Reply:
Sounds good, I will update that